app/api/data-fetching/routes.ts

```diff
@@ -0,0 +1,51 @@
+import { IAPINotRespondingError } from '#models/api-not-responding';
+import { getCarteProfessionnelleTravauxPublic } from '#models/espace-agent/carte-professionnelle-travaux-publics';
+import { EScope } from '#models/user/rights';
+
+const NotImplementedHandler = () => {
+  throw new Error('Not implemented');
+};
+
+export const APIHandler = {
+  'espace-agent/carte-professionnelle-TP': getCarteProfessionnelleTravauxPublic,
+  'espace-agent/conformite': NotImplementedHandler,
+  'espace-agent/opqibi': NotImplementedHandler,
+  'espace-agent/qualibat': NotImplementedHandler,
+  'espace-agent/qualifelec': NotImplementedHandler,
+  'espace-agent/rcs-mandataires': NotImplementedHandler,
+  'espace-agent/rne/documents': NotImplementedHandler,
+  rne: NotImplementedHandler,
+  association: NotImplementedHandler,
+  'verify-tva': NotImplementedHandler,
+  geo: NotImplementedHandler,
+  stats: NotImplementedHandler,
+} as const;
+
+export type APIPath = keyof typeof APIHandler;
+
+export const APIRoutesScope: Record<APIPath, false | EScope> = {
+  'espace-agent/carte-professionnelle-TP':
+    EScope.carteProfessionnelleTravauxPublics,
+  'espace-agent/conformite': EScope.conformite,
+  'espace-agent/opqibi': EScope.protectedCertificats,
+  'espace-agent/qualibat': EScope.protectedCertificats,
+  'espace-agent/qualifelec': EScope.protectedCertificats,
+  'espace-agent/rcs-mandataires': EScope.mandatairesRCS,
+  'espace-agent/rne/documents': EScope.documentsRne,
+  rne: false,
```

Comment:
Could it make sense to merge `APIRoutesScope` and `APIHandler` ojects ?

Something in the likes : 
```
{
	[route:string] : {
		handler: APIHandler
		scope: EScope
	}
}
```

Reply:
I split the two because I'm not sure tree-shaking would work with a single object, and we need scope client side.